audify/readers/ebook.py

```
@@ -99,25 +104,221 @@ def __init__(
+                    href = getattr(nav_point, "href", None)
+                    if href:
+                        hrefs.append(href)
+                    _walk(sub)
+                else:
+                    href = getattr(entry, "href", None)
+                    if href:
+                        hrefs.append(href)
+
+        try:
+            _walk(raw)
+        except (TypeError, Exception):
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)